x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.tsx

```diff
@@ -182,8 +182,7 @@ const AlertSummaryViewComponent: React.FC<{
   const isEndpointAlert = useMemo(() => {
     return endpointAlertCheck({ data });
   }, [data]);
-
-  const agentId = useMemo(() => {
+  const endpointId = useMemo(() => {
```

Comment:
I think you need to update this reference below.  I see that you changed `agentId` here to `endpointId`, but the reference on ln 196 below still refers to `agentId`